### ISSUE TITLE
Google Fonts don't work online

### ISSUE THREAD

**Issue by MasterOfTheTiger** [heb12/heb12-desktop#30]
**Describe the bug**
The Google fonts in the fonts settings are unavailable offline. 

**To Reproduce**
1. Turn off internet connection
2. Click on settings
3. Change fonts to one of the Google fonts
4. See that the Google fonts display Times

**Expected behavior**
I expected the Google fonts to work offline, since I had them downloaded in the /app/fonts directory (but still that pulls from an external source).

**Desktop (please complete the following information):**
 - Operating System and version (i.e Windows 10, Ubuntu 18.04): Not applicable
- Heb12 Desktop version (open the info menu): Master branch

**Solution**
I think we should find a new way to implement fonts.

**Comment by MasterOfTheTiger** [comment]
I tried that once. But I found out the way I did it did not actually download it. It will take some work to do that. If someone wants to do so, that's great, but it might be better just to download more open fonts. I don't even know if downloading them and distributing them in Heb12 is legal.

**Comment by edxtech** [comment]
YES. All fonts on Google Fonts ARE Open source! https://fonts.google.com/about

"All the fonts in our catalogue are free and open source, making beautiful type accessible to anyone for any project. This means you can share favourites and collaborate easily with friends and colleagues. Google Fonts takes care of all the licensing and hosting, ensuring that the latest and greatest version of any font is available to everyone."

For somebody all about open source, you should know that. 🤦‍♂️ 

FYI, if you are talking about official google fonts, product sans is the only one, you can't do that as it is copyrighted.

To download a google font go to [fonts.google.com](https://fonts.google.com) and click + on the fonts you want. Then open the selection menu and press download. You can then load the fonts into CSS...

**Comment by MasterOfTheTiger** [comment]
> YES. All fonts on Google Fonts ARE Open source! fonts.google.com/about

That's good. That means that it is legal for this. I just haven't had time to check into it.
